custom_components/waste_collection_schedule/waste_collection_schedule/source/publidata_ca.py

```diff
@@ -0,0 +1,482 @@
+import calendar
+import re
+from datetime import datetime, timedelta, timezone
+
+import requests
+from dateutil.rrule import (
+    DAILY,
+    FR,
+    MO,
+    MONTHLY,
+    SA,
+    SU,
+    TH,
+    TU,
+    WE,
+    WEEKLY,
+    rrule,
+    rruleset,
+)
+from waste_collection_schedule import Collection
+from waste_collection_schedule.exceptions import SourceArgumentException
+
+TITLE = "Publidata (Canada) generic source"
+DESCRIPTION = "Publidata is a Canadian public operator. Check if your area is concerned on their website."
+URL = "https://www.publidata.ca"
+COUNTRY = "ca"
+
+TEST_CASES = {
+    "Sainte-Julie City Hall": {
+        "address": "1580 Chem. du Fer-à-Cheval, Sainte-Julie, QC J3E 0A2",
+    },
+    "Calixa-Lavallée City Hall": {
+        "address": "771 Chem. de la Beauce, Calixa-Lavallée, QC J0L 1A0"
+    },
+    "Contrecoeur City Hall": {
+        "address": "5000 Rte Marie-Victorin, Contrecoeur, QC J0L 1C0"
+    },
+    "Saint-Amable City Hall": {
+        "address": "575 Rue Principale, Saint-Amable, QC J0L 1N0"
+    },
+    "Varennes City Hall": {
+        "address": "175 Rue Sainte-Anne, Varennes, QC J3X 1R6"
+    },
+    "Verchères City Hall" : {
+        "address": "581 QC-132, Verchères, QC J0L 2R0"
+    }
+
+}
+
+ICON_MAP = {
+    "omr": "mdi:trash-can",
+    "emb": "mdi:recycle",
+    "enc": "mdi:truck-remove",
+    "dv": "mdi:leaf",
+    "bio": "mdi:compost",
+    "verre": "mdi:bottle-wine",
+    "sapin": "mdi:pine-tree-variant"
+}
+
+LABEL_MAP = {
+    "omr": "Ordures ménagères",
+    "emb": "Recyclage",
+    "enc": "Volumineux",
+    "dv": "Feuilles",
+    "bio": "Compost",
+    "verre": "Verres",
+    "sapin": "Sapin de Noël",
+}
+
+HOW_TO_GET_ARGUMENTS_DESCRIPTION = {
+    "en": "Your address / Votre adresse",
+}
```

Comment:
Please remove this. This will be shown in the configuration GUI header and does not need to be there. Only use this if there is actually something useful to fill in to the header

Reply:
Done